### DISCUSSION TITLE
Station G2 / unable to flash

### DISCUSSION THREAD

**Discussion by chairmanfaust** [meshtastic/Discussions#88] [ESP32]
I feel like im going insane because this used to work fine, i had to rebuild my win11 box and reinstall the serial drivers. I plug in my station G2, i get the USB serial device (com6). I can connect with the web client to mess with the settings but when i try to flash it hangs on connecting. When i try the CLI it says:

'Failed to connect to Espressif device: No serial data received. A fatal error occured: Could not connect to an Espressif device on any of the 1 available serial ports'

I'm having similar problems on a debian box as well. I've tried different cables and ports, its probably something stupid so im just looking for suggestions

**Comment by AA7US** [comment]
You are putting the Station G2 in "Firmware Download" mode first, right?

You also didn't mention baud rate.

In the Meshtastic web Flasher, do you have the option to change the baud rate on the flash configuration screen?  If you do, try 921600 baud.  That works for me on my Station G2.

**Reply by chairmanfaust** [reply]
this was the answer... not sure how i missed this part... now i need to figure out how I'm supposed to access this for firmware updates when its on a pole at the peak of my roof... any way to keep it in 'firmware download mode'?

**Reply by thebentern** [reply]
You can try to use the 1200bps reset button on the web flasher. It does not work for everyone on every platform, but it is designed to switch the device into bootloader mode.

**Reply by chairmanfaust** [reply]
aha! that looks like it worked! awesome, back in biz....  off hand... do you happen to know if there is a CLI command for that?

**Reply by thebentern** [reply]
There is actually. `meshtastic --enter-dfu`. This should work for both NRF52 and RP2040 based boards

**Reply by chairmanfaust** [reply]
so it doesnt appear that the enter-dfu mode works for the station g2, is there a CLI equivalent for the web flasher 1200bps reset button? that works

**Reply by madeofstown** [reply]
> so it doesnt appear that the enter-dfu mode works for the station g2, is there a CLI equivalent for the web flasher 1200bps reset button? that works

where `/dev/ttyACM0` is the USB device connected to the G2

```stty -F /dev/ttyACM0 1200 && echo -e "\x00" > /dev/ttyACM0```
